src/CountBase.ts

```diff
@@ -149,3 +149,17 @@ export async function createCountBase<K>(
 
   return openCountBase(disklet, dbName)
 }
+
+export async function createOrOpenCountBase<K>(
```

Comment:
Deprecate this API in favor of just using `createCountBase` by changing the create function to not throw if database exists, but rather return the database.

Reply:
We shouldn't have all baselet types share the same disk namespace. There should never be an issue where a baselet's `databaseName` is of the wrong type when accessing it.

Reply:
Later PR for all data types